lambda_functions/v1/functions/lpa_codes/app/api/database.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+import boto3
+
+from .helpers import custom_logger
+
+logger = custom_logger()
+
+
+def db_connection():
```

Comment:
needs to be changed to function from last pr.

Reply:
Done